template/turf-template/package.json

```diff
@@ -0,0 +1,74 @@
+{
+  "name": "turf-template",
+  "version": "1.0.0",
+  "description": "Turf.js documentation",
+  "author": "Rowan Winsemius <rowanwins@yahoo.com.au>",
```

Comment:
Do you want robots sending you junk mail?

Reply:
yes please - I love it!

Reply:
🤣